astropy/io/fits/column.py

```diff
@@ -117,15 +117,17 @@
 # D: Double-precision Floating Point with exponential
 #    (E but for double precision)
 # G: Double-precision Floating Point, may or may not show exponent
-TDISP_FMT_DICT = {'I' : '{{:{width}d}}',
-                  'B' : '{{:{width}b}}',
-                  'O' : '{{:{width}o}}',
-                  'Z' : '{{:{width}x}}',
-                  'F' : '{{:{width}.{precision}f}}',
-                  'G' : '{{:{width}.{precision}g}}'}
+TDISP_FMT_DICT = {
+    'I': '{{:{width}d}}',
```

Comment:
I'm not very keen with this black type reformatting, what was wrong with the original one (beyond the extra spaces before the `:`s? I think it was way more readable.

Reply:
I find the new one much more readable ;), it makes it easier to reorganize the lines, insert at the beginning or append, etc.

Reply:
yeah, to me it looses the context, not clear with a quick glance that  it  logically belongs to the `{` and not in a function/if/else/etc structure.